documentation/src/main/asciidoc/topics/ref_redis_commands.adoc

```diff
@@ -290,6 +289,8 @@ the element. The operation is not atomic, it could remove the element from sourc
 
 * link:https://redis.io/commands/sscan[SSCAN]
 
+* link:https://redis.io/commands/lcs[STRALGO LCS]
```

Comment:
LCS is already present in the list

Reply:
That's because we implement both commands. The older and removed `STRALGO LCS` and the newer `LCS`. This URL is closest to the original STRALGO documentation, though.